components/aws/sagemaker/test/integration/utils/sagemaker_utils.py

```diff
@@ -0,0 +1,3 @@
+def describe_training_job(client, training_job_name):
```

Comment:
Can we add this code in component `_utils.py` use it from there ?

Reply:
We are going to add lots of more methods here which might be unused code in production so I think its better to keep test code separate